website/blog/2023-07-12-release-1.2.md

```diff
@@ -0,0 +1,166 @@
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/436777/251759557-bd15a631-93ee-4383-a81c-8ef3934dfb59.mp4" />
+
+### Troubleshooting page
+
+Developing an extension for Podman Desktop? Want to view the logs of Podman Desktop as well as ping your container connection? We now have a troubleshooting page!
+
+Click on the lightbulb button on the bottom right to access the page.
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/436777/248210601-e0a5deb0-44ad-4eea-9b24-134754fede80.mp4" />
+
+### Protocol handler support
+
+Podman Desktop now supports protocol handling when using the terminal! Want to access your favourite extension directly from a script or the terminal? If you type in `open podman-desktop:extension/redhat.openshift-local` in the terminal, Podman Desktop will automatically load up to the correct extension.
+
+<ReactPlayer playing controls url="https://user-images.githubusercontent.com/436777/243304511-b11ad1e4-4c2f-455c-957a-01653d2a93c8.mp4" />
+
+---
+
+## Other Notable Features
+
+- Background colors and FormPage ([PR #2977](https://github.com/containers/podman-desktop/pull/2977))
+- Add ability to add insecure registry / skipping cert verify ([PR #2896](https://github.com/containers/podman-desktop/pull/2896))
+- Add support for icon contribution ([PR #2984](https://github.com/containers/podman-desktop/pull/2984))
+- Add warning dialog message that virtual machine has low memory limit ([PR #2822](https://github.com/containers/podman-desktop/pull/2822))
+- Propose indexed name for new pod ([PR #3028](https://github.com/containers/podman-desktop/pull/3028))
+- Add restart button after enabling / disabling mac os compatibility ([PR #2841](https://github.com/containers/podman-desktop/pull/2841))
+- Add environment related helper constants ([PR #3079](https://github.com/containers/podman-desktop/pull/3079))
+- Allow entrypoint and cmd when starting container ([PR #3031](https://github.com/containers/podman-desktop/pull/3031))
+- Add a way to debug stores in troubleshooting page ([PR #3121](https://github.com/containers/podman-desktop/pull/3121))
+- Add custompick component (#2855) ([PR #3012](https://github.com/containers/podman-desktop/pull/3012))
+- Dynamic breadcrumbs ([PR #3119](https://github.com/containers/podman-desktop/pull/3119))
+- Icons on form pages ([PR #3155](https://github.com/containers/podman-desktop/pull/3155))
+- Switch more pages to formpage ([PR #3162](https://github.com/containers/podman-desktop/pull/3162))
+- Add rename image button ([PR #2588](https://github.com/containers/podman-desktop/pull/2588))
+- Fixed headers, improved scrollbars ([PR #2863](https://github.com/containers/podman-desktop/pull/2863))
+- Add support for protocol handler 'podman-desktop:' ([PR #2735](https://github.com/containers/podman-desktop/pull/2735))
```

Comment:
I think you can delete the lines for:
- protocol handler PR (already listed above)
- release notes 1.1 (not notable)
- one of the Kube context lines (it's duplicated).